### ISSUE TITLE
Avalonia: loosing lines, only seeing points (window size, zooming)

### ISSUE THREAD

**Issue by Limula-PMA** [ScottPlot/ScottPlot#3765]
**Issue:**
A plot with a couple of signals can (depending on the number of points and graph size) suddenly no longer render the lines between the points.

https://github.com/ScottPlot/ScottPlot/assets/150119089/2c670702-0d28-409f-a51e-71c41909b5dc

I don't know if I forgot something (it looks like this issue was introduced in recent releases, because I don't believe I saw this behavior with ScottPlot 5.0.23 and earlier.

**ScottPlot Version:** 
ScottPlot.Avalonia 5.0.32
Avalonia 11.0.10

**Comment by swharden** [comment]
Thanks for reporting this @Limula-PMA! Can you share some code we can use to reproduce this issue? Is it a scatter plot or a signal plot? How many points of data are being plotted? More information that makes it easier for us to recreate the issue would help in arriving at a fix!

**Comment by Limula-PMA** [comment]
Hello, 

My apologies for the lack of details. 
I tried to reproduce it with a simple project, and all you need is one signal with enough points :

```cs
var rand = new Random();
UxPlot.Plot.Add.Signal(Enumerable.Range(0, 1000).OrderBy(_ => rand.Next()).ToList());
```

Then, if you resize (or zoom in) you will eventually get only the points:

https://github.com/ScottPlot/ScottPlot/assets/150119089/eb8e43c2-afd5-42eb-9fbc-1b0bc3debcfb

**Comment by swharden** [comment]
Thanks for reporting this @Limula-PMA! It seems this issue is related to low density display mode for lists, but arrays work as expected...

```cs
// broken
List<int> values = Enumerable.Range(0, 1000).OrderBy(_ => Random.Shared.Next()).ToList();
formsPlot1.Plot.Add.Signal(values);
```

```cs
// works
int[] values = Enumerable.Range(0, 1000).OrderBy(_ => Random.Shared.Next()).ToArray();
formsPlot1.Plot.Add.Signal(values);
```

**Comment by swharden** [comment]
Fixed in e613a842a5fc616033062d56bdf96d8d99f74e2d and will be in the next release! Thanks again for reporting this 👍
